handlers/requestHandlers.js

```diff
@@ -1,47 +1,62 @@
-"use strict"
+"use strict";
 
-let documentToObject = require('../restit');
+let cached,
+    ttl = (process.env.CACHE_TTL || 60) * 1000;
```

Comment:
also can we make the default smaller than 60s. Make let it be 1sec or something really small.